src/value.rs

```diff
@@ -223,6 +225,12 @@ impl UserData for Color3uint8Value {
 #[derive(Debug, Clone, Copy)]
 pub struct Vector3Value(Vector3);
 
+impl Into<Vector3> for Vector3Value {
+    fn into(self) -> Vector3 {
+        self.0
+    }
+}
+
```

Comment:
I think it would be better to introduce a dedicated `inner()` method for these wrapper types to make code using these conversions a little clearer. Alternatively, we could just define them as `pub struct Vector3Value(pub Vector3)` and whatnot and just have code use `.0`.

Reply:
I went with the inner method 😄